hassio-google-drive-backup/backup/static/js/theme.js

```diff
@@ -147,206 +127,58 @@ class Color {
-      'border-bottom': `1px solid ${accent.toCss()}`,
-      '-webkit-box-shadow': `0 1px 0 0 ${accent.toCss()}`,
-      'box-shadow': `0 1px 0 0 ${accent.toCss()}`,
-    },
-    '.card': {
-      'background-color': background.toCss(),
-      'box-shadow': `0 2px 2px 0 ${shadow1.toCss()}, 0 3px 1px -2px ${shadow2.toCss()}, 0 1px 5px 0 ${shadow3.toCss()}`,
-    },
-    'nav a': {
-      'color': accentText.toCss(),
-    },
-    '.btn, .btn-large, .btn-small': {
-      'color': accentText.toCss(),
-    },
-    '.bmc-button': {
-      'line-height': '15px',
-      'height': '25px',
-      'text-decoration': 'none',
-      'display': 'inline-flex',
-      'background-color': background.toCss(),
-      'border-radius': '3px',
-      'border': '1px solid transparent',
-      'padding': '3px 2px 3px 2px',
-      'constter-spacing': '0.6px',
-      'box-shadow': `0px 1px 2px ${shadowBmc.toCss()}`,
-      '-webkit-box-shadow': `0px 1px 2px 2px ${shadowBmc.toCss()}`,
-      'margin': '0 auto',
-      'font-family': "'Cookie', cursive",
-      '-webkit-box-sizing': 'border-box',
-      'box-sizing': 'border-box',
-      '-o-transition': '0.3s all linear',
-      '-webkit-transition': '0.3s all linear',
-      '-moz-transition': '0.3s all linear',
-      '-ms-transition': '0.3s all linear',
-      'transition': '0.3s all linear',
-      'font-size': '17px',
-    },
-    '.bmc-button span': {
-      'color': text.toCss(),
-    },
-    ':root': {
-      '--cls-color': text.toCss(),
-      '--cls-sec-color': text.toCss(),
-      '--cls-size': '2rem',
-      '--cls-margin': '1rem',
-      '--cls-speed': '4s',
-    },
-    'textarea': {
-      'color': text.toCss()
-    },
-    '.error-card-container': {
-      'color': text.toCss(),
-      'background-color': background.toCss(),
-      'margin': '4px',
-      'box-shadow': `0 2px 2px 0 ${shadow1.toCss()}, 0 3px 1px -2px ${shadow2.toCss()}, 0 1px 5px 0 ${shadow3.toCss()}`,
-      'padding': '3px 2px 3px 2px',
-    },
-    '.error-card-container  a': {
-      'color': linkAccent.toCss(),
-    },
-    '.error-card-container .ha-blue': {
-      'background-color': accent.toCss(),
-      'color': accentText.toCss(),
+    ":root": {
```

Comment:
Now, all custom CSS is in `static.css`; and this function only sets the CSS variables.

Reply:
Yeah! And it is actually what Home Assistant also uses ;)